pymc/logprob/transforms.py

```diff
@@ -617,6 +617,25 @@ def measurable_special_exp_to_exp(fgraph, node):
         return [1 / (1 + pt.exp(-inp))]
 
 
+@node_rewriter([pow])
+def measurable_power_expotent_to_exp(fgraph, node):
+    """Convert power(base, x) of `MeasurableVariable`s to exp form."""
+    base, inp_exponent = node.inputs
+
+    # if base is measurable then it is power(x, exponent) and not power(base, x) as expected
+    # and should use PowerTransform
```

Comment:
Trying to clarify a bit more:
```suggestion
    # When the base is measurable we have `power(rv, exponent)`, which should be handled by `PowerTransform` and needs no further rewrite.
    # Here we change only the cases where exponent is measurable `power(base, rv)` which is not supported by the `PowerTransform`
```